src/Button/Button.less

```diff
@@ -0,0 +1,75 @@
+.Button {
+  margin: 0 8px;
+  background: #ffffff;
+  border: 0;
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+  //border: 1px solid #4274F6;
+  //box-shadow: 0 2px #E9F1FE;
+  // color: #4274F6;
+  &[disabled]{
+    color: #4B4B59;
+    background-color: #D7D9D9;
+    border-bottom: 3px solid #A4A6A6;
+  }
+
+}
+
+.Button--primary {
+  background-color: #4274F6;
+  color: #FAFCFC;
+  border-bottom: 3px solid #345CC4;
+  //box-shadow: 0 3px #b1cbf2;
+  &:hover,
+  &:focus,
+  &:active {
+    //outline: 0;
```

Comment:
This will get rid of the blue border when you click/focus, which is fine in terms of accessibility since we're also changing the color. We should probably add a different active state since there isn't any feedback when you actually click, but I'll leave that to carl